src/sklearn_evaluation/util.py

```diff
@@ -81,8 +81,14 @@ def fn(obj):
     return fn
 
 
-def _sorted_map_iter(d):
-    ord_keys = sorted(d.keys())
+def _sorted_map_iter(d, sort=True):
+    if sort:
+        try:
+            ord_keys = sorted(d.keys())
+        except TypeError:
+            raise TypeError('Sorting not supported. Plase pass sort=False')
```

Comment:
python doesn't like when we raise exceptions inside an `except` block, please change this to a chained exception